applications/CoSimulationApplication/python_scripts/solver_wrappers/kratos/kratos_base_wrapper.py

```diff
@@ -98,8 +102,25 @@ def _CreateAnalysisStage(self):
 
     def __GetAnalysisStage(self):
         if self.settings["solver_wrapper_settings"].Has("analysis_stage_module"):
-            analysis_stage_module = import_module(self.settings["solver_wrapper_settings"]["analysis_stage_module"].GetString())
-            return analysis_stage_module.Create(self.model, self.project_parameters)
+            module_name = self.settings["solver_wrapper_settings"]["analysis_stage_module"].GetString()
+            analysis_stage_module = import_module(module_name)
+            if hasattr(analysis_stage_module, "Create"):
+                return analysis_stage_module.Create(self.model, self.project_parameters)
+            else:
+                KM.Logger.PrintWarning("KratosBaseWrapper", f'The analysis_stage_module "{module_name}" does not have a "Create" function, trying to create the AnalysisStage directly...')
+                if self.settings["solver_wrapper_settings"].Has("analysis_name"):
+                    analysis_stage_name = self.settings["solver_wrapper_settings"]["analysis_name"].GetString()
```

Comment:
```suggestion
                    analysis_stage_name = self.settings["solver_wrapper_settings"]["analysis_stage_name"].GetString()
```
For consistency with the `analysis_stage_module` field. Besides, as I pointed before, in our current ProjectParameters.json this field is named `analysis_stage`. I think it would be good to use the same key.

Reply:
Please, note my comment. I think we should agree towards standarizing this with current `ProjectParameters.json`.

Reply:
Yes, but the current key was already there, if @philbucher agrees I will change it

Reply:
OK. Now I see. You meant changing `analysis_stage_module`. If this is the case, the discussion is not that easy. Lets leave it as it is for the moment.

Reply:
(resolving again)